lib/file_database.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'json'
+
+class FileDatabase
+  def initialize(file_dir)
+    @file = File.open(file_dir, 'r')
```

Comment:
heh, sorry I didn't explain myself well :sweat_smile: :sweat_drops: 

when I wrote this in the ticket I meant this is what you would use in your CLI app (`bin/app`)
so it was more like "tell me what mode you would use in the chat" rather than "code it"

in the `bin/app` you would have as you have correctly written:
```ruby
file = File.open(file_dir, 'r')
database = FileDatabase.new(file)
```

but in the constructor you just need:
```ruby
  def initialize(file)
    @file = file
  end
```

everything else is fine. It's just that we are not yet updating the `bin/app`.